core-web/libs/sdk/experiments/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "sdk-experiments",
```

Comment:
```suggestion
  "name": "@dotcms/experiments",
```

Reply:
ohh, I followed the sdk-client... I saw now the other 2 use that pattern. I will change it.

Reply:
This pattern is what will do that when this library gets published it goes into our npm organization

Reply:
Fixed.